### DISCUSSION TITLE
What about a Amazon Echo version?

### DISCUSSION THREAD

**Discussion by eugeneniemand** [justLV/onju-voice#60] [Ideas]
I have 9 Echo dots (3rd gen) and two Echo's and was hoping to use something a bit more local. Does anyone know of plans to support Echo or a similar product/projects to this for Echo?

**Comment by Hedda** [comment]
See these two indirectly related requests where some other PCB developers are discussion creating new alternative PCBs for Google Mini's:

* https://github.com/justLV/onju-voice/issues/59

* https://github.com/justLV/onju-voice/issues/74

Also see existing feature request issue and discussion that was closed by original developer for this project has no plans on doing this himself:

* https://github.com/justLV/onju-voice/issues/36

In summery; someone else needs to step up and make this and all of those happen for it to become a reality.

PS: FYI, for reference, see related requests closed as "not planed" by the original developer: 

* https://github.com/justLV/onju-voice/issues?q=is%3Aissue+is%3Aclosed+nest
  * https://github.com/justLV/onju-voice/issues/27  
  * https://github.com/justLV/onju-voice/issues/28
  * https://github.com/justLV/onju-voice/issues/29
  * https://github.com/justLV/onju-voice/issues/36
